### PR TITLE
Add regular Nyuki checks

### PR DESCRIPTION
**Pull request by nirvn**
Say hello to nyuki, he'll watch over you.

### CODE REVIEW

**Comment by qfield-fairy** [comment]
🎉 Ta-daaa, freshly created APKs are available for 8f791b50ff0b61bda9acee8cfd5377cca2958c20: [**arm64-android**](https://sos-ch-dk-2.exo.io/qfieldapks/ci-builds/qfield-dev-3413-5432d13f79d0bad468d0ecfe7b7d3c0278cede4b-arm64-android.apk)
<details>
<summary>Other architectures</summary>

- [arm-android](https://sos-ch-dk-2.exo.io/qfieldapks/ci-builds/qfield-dev-3413-5432d13f79d0bad468d0ecfe7b7d3c0278cede4b-arm-android.apk)
- [x64-android](https://sos-ch-dk-2.exo.io/qfieldapks/ci-builds/qfield-dev-3413-5432d13f79d0bad468d0ecfe7b7d3c0278cede4b-x64-android.apk)
- [x86-android](https://sos-ch-dk-2.exo.io/qfieldapks/ci-builds/qfield-dev-3413-5432d13f79d0bad468d0ecfe7b7d3c0278cede4b-x86-android.apk)
</details>

File: src/qml/qgismobileapp.qml [hunk 1]
```diff
@@ -3094,6 +3094,108 @@ ApplicationWindow {
...
+        PropertyChanges {
+          target: nyuki
+          anchors.bottomMargin: -200
+        }
+      }
+    ]
+
+    transitions: Transition {
+      from: "*"; to: "*"
+      NumberAnimation { target: nyuki; property: "anchors.bottomMargin"; duration: 1000; easing.type: Easing.InOutQuad }
```
Comment (suricactus) [inline src/qml/qgismobileapp.qml:3195]: Can we have it multiline?

File: src/qml/qgismobileapp.qml [hunk 2]
```diff
@@ -3094,6 +3094,108 @@ ApplicationWindow {
     visible: false
   }
 
+
+  Image {
```
Comment (suricactus) [inline src/qml/qgismobileapp.qml:3098]: Wondering whether it makes sense to move the whole thing in a separate component file?

**Review by suricactus — APPROVED** [review]
Loooks veeery nice, great job!

File: src/qml/qgismobileapp.qml (hunk 2 above)
Comment (nirvn) [reply src/qml/qgismobileapp.qml:3098]: I'm worried having it in a separate component file will make it too easy to find out :)

File: src/qml/LocatorItem.qml [hunk 3]
```diff
@@ -12,6 +12,9 @@ Item {
   property bool searchFieldVisible: searchField.visible
   property alias locatorModelSuperBridge: locator
 
+  signal searchTermChanged(var searchTerm)
```
Comment (m-kuhn) [inline src/qml/LocatorItem.qml:15]: Can you document this signal (and especially mention the connection to `serchTermHandled`?

File: src/qml/qgismobileapp.qml (hunk 2 above)
Comment (m-kuhn) [reply src/qml/qgismobileapp.qml:3115]: If someone looks at the source I think he deserves to know

**Review by m-kuhn — APPROVED** [review]

**Pull request merged**